CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # Changelog
-## Unreleased
+## [v3.1.0] - 2021.12.03
```

Comment:
why this file is contained in the PR?
It's from branch which is already merged in master. This applies to the other files as the image file or web components scripts.
I guess this PR should contain changes in only 3 files so it must be rebased wrongly
![image](https://user-images.githubusercontent.com/4511883/146365438-8dea1aa7-80b0-427d-951f-417e22a09239.png)

why these commits from master are in between of the commits of this branch?